### PR TITLE
Make devtag compatible with pkgdown and more compatible with roxygen2

### LINKED ISSUE #1: Review feedback from Maelle on this day
on Mattermost

In particular check how it plays out with pkgdown

If packagedown doesn't remove the files provide a wrapper that does remove them

### PR DESCRIPTION
**Pull request by maelle**
Fix #1

### CODE REVIEW

**Comment by maelle** [comment]
igraph PR https://github.com/igraph/rigraph/pull/1507

File: R/roxygen.R [hunk 1]
```diff
@@ -27,14 +27,30 @@ roclet_output.roclet_dev <- function(x, results, base_path, ...) {
...
   dev_topics <- sapply(dev_blocks, function(x) x$object$topic)
-  dev_files <- if (length(dev_topics)) file.path("^man", paste0(dev_topics, "\\.Rd$"))
+
+  if (length(dev_topics) == 0) {
+    return(invisible())
+  }
+
+  # .Rbuildignore
   ignored <- readLines(".Rbuildignore")
+  dev_files <- file.path("^man", paste0(nice_name(dev_topics), "\\.Rd$"))
```
Comment (maelle) [inline R/roxygen.R:37]: I noticed this because I have a topic called `.thing` whose Rd file is `dot-thing.Rd` :sweat_smile:

File: README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ pak::pak("moodymudskipper/devtag")
 To use {devtag} :
 
 1)  Use “@dev” for unexported functions where you would have used
-    “@export” for exported functions.
+    “@export” for exported functions. Do not use “@rdname”.
```
Comment (maelle) [inline README.md:31]: because otherwise, our logic for finding the file no longer works

**Comment by maelle** [comment]
@moodymudskipper friendly reminder :smile_cat:

**Comment by moodymudskipper** [comment]
thanks!

**Pull request merged**